### PR TITLE
Fix setup.sh so it does what it should

### PR DESCRIPTION
**Pull request by pstray**
Remove cd (which makes the rest of the script fail)
    Remove space around = and replace `` with $()

### CODE REVIEW

**Comment by KonstantinSchubert** [comment]
+1 
I wonder on which system this was tested and worked?

File: setup.sh [hunk 1]
```diff
@@ -4,8 +4,7 @@ apt-get update
 apt-get install  libasound2-dev memcached python-pip mpg123 python-alsaaudio
 pip install -r requirements.txt
 cp initd_alexa.sh /etc/init.d/alexa
-cd /etc/rc5.d
-ln -s ../init.d/alexa S99alexa
+ln -s ../init.d/alexa /etc/rc5.d/S99alexa
```
Comment (wajjs) [inline setup.sh]: this link would be incorrect.  you should change ../init to /etc/init

Comment (pstray) [inline setup.sh]: No it won't, symlinks are relative to where the are not from where you make
them. Take a look at any og the other symlinks in rc5.d and you will notice
most of them use ../init.d and not /etc/init.d.

Comment (wajjs) [inline setup.sh]: > , symlinks are relative to where the are not from where you make them. 

Ok. I did not know that.

**Comment by sammachin** [comment]
superseded by later PR, sorry I'm closing this to tidy things up and I'll have a fresh look at the setup scripts in the next version

**Pull request closed without merging**